data/filedefs/filetypes.autoit

```diff
@@ -0,0 +1,98 @@
+# For complete documentation of this file, please see Geany's main documentation
+# AutoIt 3.3.14.2
+[styling]
+# Edit these in the colorscheme .conf file instead
+default=default
+comment=comment
+commentblock=comment
+number=number,bold,italic
+function=function,bold,italic
+keyword=type
+macro=preprocessor,bold
+string=string
+operator=operator,bold
+variable=keyword_2
+sent=0xd77100;;true;false
+preprocessor=preprocessor,italic
+special=type
+expand=default
+comobj=keyword_1,italic
+udf=class,italic
```

Comment:
styles here shouldn't include custom colors, italics, etc.  I know it can be tricky to chose the mappings for things that don't have direct equivalents, but having arbitrary tuned ones makes it virtually impossible to theme.